### DISCUSSION TITLE
how to use teams?

### DISCUSSION THREAD

**Discussion by matiazar** [spatie/laravel-permission#2692] [Q&A]
Hi
I wanna create a role based teams like
client 1: admin and users
client 2: admin and users

how can I use team functionality? do I need to install any team package? which one?

thanks in advance

**Comment by parallels999** [comment]
[laravel-permission/v6/teams-permissions](https://spatie.be/docs/laravel-permission/v6/basic-usage/teams-permissions)

**Reply by parallels999** [reply]
[spatie/laravel-permission/database/migrations/create_permission_tables.php](https://github.com/spatie/laravel-permission/blob/main/database/migrations/create_permission_tables.php.stub)

> I need to create any other table like "teams"

Yes

> install another external package for this to work.

No

**Comment by matiazar** [comment]
hi @parallels999 thanks for answering me
yes I saw that.. I understand how to activate it... but I dont understand if I need to create any other table like "teams" table or install another external package for this to work.

how can I make a user part of a team... and this type of things..
